strawberry_django/fields/types.py

```diff
@@ -340,25 +352,81 @@ def resolve_model_field_type(
     model_field: Union[Field, reverse_related.ForeignObjectRel],
     django_type: "StrawberryDjangoDefinition",
 ):
-    is_relay = issubclass(django_type.origin, relay.Node)
-    model_field_type = type(model_field)
-    field_type: Any = None
+    settings = django_settings()
+
+    # Django choices field
+    if (
+        TextChoicesField is not None
+        and IntegerChoicesField is not None
+        and isinstance(
+            model_field,
+            (TextChoicesField, IntegerChoicesField),
+        )
+    ):
```

Comment:
I have the feeling we could migrate these if to call a function, to make it clearer what they do 😊

same for the content, but we can do that in another PR